docs/infra-providers/digitalocean/README.md

```diff
@@ -0,0 +1,211 @@
+# llm-d on DigitalOcean Kubernetes Service (DOKS)
```

Comment:
Be sure to update `guides/prereq/infrastructure/README.md` to add the DOKS link in alphabetical order (after coreweave and before GKE)